src/unlimited.cpp

```diff
@@ -1533,7 +1612,7 @@ static const CRPCCommand commands[] =
 #ifdef DEBUG
     { "util",               "getstructuresizes",      &getstructuresizes,      true  },  // BU
 #endif
-
+    { "util",               "log",                    &setlog,                 true  }, //VDH
```

Comment:
please don't put your initials next to code

Reply:
k